.gitmodules

```diff
@@ -8,3 +8,6 @@
 	path = lib/sdl/SDL2
 	url = https://github.com/XboxDev/nxdk-sdl.git
 	branch = nxdk
+[submodule "lib/pdclib"]
+	path = lib/pdclib
+	url = https://github.com/XboxDev/pdclib-nxdk.git
```

Comment:
please move to https://github.com/XboxDev/nxdk-pdclib.git to match the other(s).

Reply:
I've ended up moving it, after @thrimbor contacted me that he didn't have permissions to access the "Settings" panel on GitHub.